08-Placing-Barriers-in-the-Simulation-Box/04-Wall-potential-MD.ipynb

```diff
@@ -0,0 +1,408 @@
+{
```

Comment:
<p>Very minor comment in the Questions section that it "molecular dynamics" is referred to by full name here but the acronym "HPMC" is used in the next section. Unimportant, but I still noticed it was inconsistent. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/glotzerlab/hoomd-examples/pull/111/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>